src/components/buttons/ButtonsContainer.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { View } from 'react-native'
+
+import { styled } from '../hoc/styled'
+import { MainButton, MainButtonType } from '../themed/MainButton'
+
+export interface ButtonContainerProps {
+  label: string
+  onPress: () => void | Promise<void>
+  disabled?: boolean
+}
```

Comment:
Call this `ButtonInfo` or something. These aren't the top-level parameters a component expects, so we can't call them `Props`.